### PR TITLE
Allow to ignore strand while sorting CpGs' position

### PR DESCRIPTION
**Pull request by Ning-L**
related to #9

### CODE REVIEW

**Comment by gabrielodom** [comment]
Hi @Ning-L, I'm sorry I missed this. Thank you so much for the hard work!
I will take a look at your code next week and (hopefully) merge your branch. I saw your comments in Issue #9, so thank you for adding thorough documentation

**Comment by Ning-L** [comment]
Hi @gabrielodom , glad to help. 
Just find a bug in `lmmTest.R` and I opened a new issue #12

File: R/lmmTest.R [hunk 1]
```diff
@@ -127,6 +130,7 @@ lmmTest <- function(betaOne_df, pheno_df, contPheno_char, covariates_char,
     ps_df <- data.frame(
       Estimate = NA_real_,
       StdErr = NA_real_,
+      Stat = NA_real_,
```
Comment (gabrielodom) [inline R/lmmTest.R:133]: Thank you for this! I can't believe I missed it :/

File: R/util1_OrderCpGsByLocation.R [hunk 2]
```diff
@@ -52,7 +55,7 @@ OrderCpGsByLocation <- function(
     )
   }
   CpGs.gr <- sort(
-    CpGlocations.gr[ CpGs_char[goodCpGs_lgl] ]
+    CpGlocations.gr[CpGs_char[goodCpGs_lgl], ignore.strand = ignoreStrand]
```
Comment (gabrielodom) [inline R/util1_OrderCpGsByLocation.R:58]: In this form, we would have an "invalid subsetting" error. Notice that the comma is inside the call to `[`. We need to make sure that the `ignore.strand` argument is being passed to the `sort()` function from `BiocGenerics::`.

**Review by gabrielodom — APPROVED** [review]
There is an error in your version of `OrderCpGsByLocation()`, but I will fix it after the merge. I've shown the updated code below. Thank you!!

```
CpGs.gr <- sort(
    CpGlocations.gr[ CpGs_char[goodCpGs_lgl] ],
    ignore.strand = ignoreStrand
  )
```

**Comment by Ning-L** [comment]
> There is an error in your version of `OrderCpGsByLocation()`, but I will fix it after the merge. I've shown the updated code below. Thank you!!
> 
> ```
> CpGs.gr <- sort(
>     CpGlocations.gr[ CpGs_char[goodCpGs_lgl] ],
>     ignore.strand = ignoreStrand
>   )
> ```

Exactly, the parameter was not in the right place. Thanks for correcting it!

**Pull request merged**